dt-metrics/contacts/milestones.php

```diff
@@ -61,7 +61,7 @@ public function scripts() {
                 'translations' => [
                     'milestones' => __( "Milestones", 'disciple_tools' ),
                     'filter_contacts_to_date_range' => __( "Filter contacts to date range:", 'disciple_tools' ),
-                    'all_time' => __( "All time", 'disciple_tools' ),
+                    'all_time' => _x( "All Time", 'Dates', 'disciple_tools' ),
```

Comment:
Just have them be all without context. All time is a string used in many places, so we are bound to have duplicates

Reply:
This was a duplicate and the only one missing the context. It won't create any new strings if this change is applied.

Reply:
You added the "dates" context to multiple of them...

Reply:
> You added the "dates" context to multiple of them. Let's remove it.

No. They were already there (as I recall) so just added to this one so it didn't get duplicated.

Reply:
See the 2 comments right below this one. You might need to click the Load more button.